src/screens/internal/ShareImage/HomeShareCardImage.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import SocialLocationPreview from 'components/SocialLocationPreview/SocialLocationPreview';
+import { ShareCardWrapper } from './ShareCardImage.style';
+import { DarkScreenshotWrapper } from './ShareImage.style';
+import { SCREENSHOT_CLASS } from 'components/Screenshot';
+import { Header } from './LocationShareCardImage';
+
+/**
+ * Screen that just shows the appropriate share card so that we can take a
+ * screenshot that we then use as our OpenGraph image.
+ */
+const HomeShareCardImage = () => {
+  // FIXME: do we need <ScreenshotReady> on this one? it was missing before
```

Comment:
It gets added by a nested component here once we've loaded the necessary data and rendered the map: https://github.com/covid-projections/covid-projections/blob/ead6033a754e31b5573b16884227f05dc298c532/src/components/Map/USACountyMap.tsx#L191

Reply:
ah, thanks : )